resources/contentful/getPersonList.graphql

```diff
@@ -0,0 +1,25 @@
+query peopleByListQuery($listId: String!){
+  peopleList(id: $listId) {
+    internalName
```

Comment:
Intenal name fetched to external application (outside contentful)? Consider field names and make them more accurate if possible.

In other data internal name is used as a contentful name where it can include details like "where" a given list component is. For example it could be "Frontpage article list". This is to help finding components in contentful's search while doing editorial work.